ietf-layer0-types.yang

```diff
@@ -1550,14 +1546,75 @@ module ietf-layer0-types {
         container explicit-mode {
           config false;
           description
-            "The set of attributes for an explicit mode";
-          // uses explicit-mode;
+            "The set of attributes for an explicit mode.
+
+            The explicit mode attributes can be defined either:
+            - as a leafref to an explicit mode template; or:
+            - using the explicit-mode grouping.
+
+            Therefore, this container shall be augmented with either
+            a use statement for the explicit-mode grouping or a
+            leafref to an explicit mode template and the choice
+            depends on where this grouping is used.
+            In the latter case, the augmentation shall provide the
+            correct XPaths that depend on where this grouping is used
+            within the YANG data tree.
+            See for example the YANG code commented below";
+/*  YANG example for using the explicit-mode grouping:
```

Comment:
Would it be better to describe this example inside the description instead of using commented YANG code which is not a common practice in standard YANG?